lib/rollbax/client.ex

```diff
@@ -118,9 +104,30 @@ defmodule Rollbax.Client do
       {:error, _} ->
         Logger.error("(Rollbax) API returned malformed JSON: #{inspect body}")
     end
+
+    %{state | hackney_responses: Map.delete(responses, ref)}
+  end
+
+  defp handle_hackney_response(ref, {:status, code, description}, %{hackney_responses: responses} = state) do
+    if code != 200 do
+      Logger.error("(Rollbax) unexpected API status: #{code}/#{description}")
+    end
+
+    %{state | hackney_responses: Map.put(responses, ref, [])}
+  end
+
+  defp handle_hackney_response(_ref, {:headers, headers}, state) do
+    Logger.debug("(Rollbax) API headers: #{inspect(headers)}")
+    state
+  end
+
+  defp handle_hackney_response(ref, body_chunk, %{hackney_responses: responses} = state)
+  when is_binary(body_chunk) do
+    %{state | hackney_responses: Map.update!(responses, ref, &[body_chunk | &1])}
```

Comment:
`Map.update!(responses, ref, &[&1 | body_chunk])` and then we can skip `Enum.reverse` but call `IO.iodata_to_binary/1` directly on it.